app/conf/rdf-regression-example.conf

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2015, Cloudera, Inc. All Rights Reserved.
+#
+# Cloudera, Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"). You may not use this file except in
+# compliance with the License. You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the
+# License.
+
+# A very basic example config file configuring only the essential elements to
+# run a decision forest-based application.
+
+# It's possible to specify reusable values:
+kafka-brokers = "b03.example.com:9092,b04.example.com:9092"
+zk-servers = "b01.example.com:2181,b02.example.com:2181/kafka"
+hdfs-base = "hdfs:///user/example/Oryx"
+
+oryx {
+  id = "RDFExample"
```

Comment:
This seems fine; maybe just change the ID of the example for completeness? You're also welcome to rename the existing RDF example (and update docs if needed).